api/proto/apps/aifunction/ai-function.proto

```diff
@@ -29,8 +29,9 @@ service AiFunction {
 
 message ApplyRequest {
   string functionName = 1;
-  Prompt prompt = 2;
+  FunctionParams functionParams = 2;
```

Comment:
Maybe use `google.protobuf.Value functionParams` with custom scene functionParam message definition is better.

Reply:
Ok